packages/office-ui-fabric-react/src/components/Calendar/Calendar.scss

```diff
@@ -119,6 +122,7 @@ $Calendar-dayPicker-margin: 10px;
   line-height: $Calendar-day;
   @include ms-font-m-plus;
   color: $ms-color-neutralPrimary;
+  border-radius: 2px;
```

Comment:
Please apply all my feedback first with rounded corners enabled to make sure the calculations work, but before committing these changes, let's send out a final iteration of the PR with the rounded corners commented out.